assets/skins/dialogScreen.skin

```diff
@@ -36,6 +36,9 @@
         },
         "ScrollableArea": {
             "background": ""
+        },
+        "UIImage": {
+            "background": "heartsIcon"
```

Comment:
I don't think we should work with a background image for the `UIImage` widget here. 

Note that the background image is rendered behind the widget itself (i.e., the background). The `UIImage` widget would render another image on top if properly configured. I think we should rather set the source for the image to render in the widget correctly.

Reply:
That sounds logical, yes, having a background image should be the way we go about. I'll proceed on removing this line when I work on this task 👍 (or maybe closing that PR will work as well 🤔)